src/Razor/test/testapps/Directory.Build.props

```diff
@@ -38,4 +38,26 @@
 
   <Import Project="After.Directory.Build.props" Condition="Exists('After.Directory.Build.props')" />
 
+  <!--
+    NOTE, BELOW
+
+    These bits are a temporary workaround until we're able to get on the latest SDK which includes the latest Roslyn bits.
+    Without them we can't properly publish testapps in a standalone fashion without investing in temporary infrastructure.
```

Comment:
This shouldn't be a temporary fix. Should _always_ test against the Roslyn version that will ship with the rest of the release.

The SDK isn't relevant in this case either. We have no plans to rev the .NET SDK faster than it is released. While a lot of our build infrastructure is about building and using what's coming in that release, the SDK itself has to stay back.

Reply:
Of course, this is blocking several repos though. Getting in this hack to unblock that.

Reply:
Filed an issue to track "doing the right thing": https://github.com/aspnet/AspNetCore/issues/12268